solr/core/src/test/org/apache/solr/cloud/TestConfigSetsAPIZkFailure.java

```diff
@@ -111,14 +110,14 @@ public void testCreateZkFailure() throws Exception {
     SolrZkClient zkClient = new SolrZkClient(solrCluster.getZkServer().getZkAddress(),
         AbstractZkTestCase.TIMEOUT, AbstractZkTestCase.TIMEOUT, null);
     try {
-      ZkConfigManager configManager = new ZkConfigManager(zkClient);
-      assertFalse(configManager.configExists(CONFIGSET_NAME));
+
+      assertFalse(solrCluster.getOpenOverseer().getCoreContainer().getConfigSetService().checkConfigExists(CONFIGSET_NAME));
```

Comment:
long and used twice.  Declare `solrCluster.getOpenOverseer().getCoreContainer().getConfigSetService()`